source/codegen/metadata/nidmm/functions.py

```diff
@@ -1747,14 +1807,15 @@
     },
     'Init': {
         'cname': 'niDMM_init',
+        'codegen_method': 'public',
         'init_method': True,
         'parameters': [
             {
                 'cppName': 'resourceName',
                 'direction': 'in',
                 'grpc_type': 'string',
                 'name': 'resourceName',
-                'type': 'ViRsrc'
+                'type': 'ViString'
```

Comment:
Note: This changed types but it shouldn't affect the underlying type in .proto or the underlying type passed into the DLL. This may be more accurate since it's just sourced from base hapigen metadata now.

Reply:
The types are interchangeable so I'm not worried.